### PR TITLE
Minor fixes to main layout

### PR DESCRIPTION
**Pull request by imin314**

### CODE REVIEW

File: src/app/components/CreditPoolHeader/components/AppButton/AppButton.style.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from 'utils/styles';
+
+export const useStyles = makeStyles(
+  () => ({
+    root: {
+      padding: 0,
+    },
+  }),
+  { name: 'appButton' },
```
Comment (sk1e) [inline src/app/components/CreditPoolHeader/components/AppButton/AppButton.style.ts]: I am using dash case (app-button) for such names, let's follow single naming scheme to keep consistency

File: src/app/components/NewHeader/NewHeader.style.ts [hunk 2]
```diff
@@ -13,6 +13,15 @@ export const useStyles = makeStyles(
       display: 'flex',
     },
 
+    logo: {
+      display: 'flex',
+
+      fontSize: theme.spacing(2.5),
+      [theme.breakpoints.up('tabletXS')]: {
+        fontSize: theme.spacing(5),
```
Comment (sk1e) [inline src/app/components/NewHeader/NewHeader.style.ts]: Was this code copied? We've come to using pixels instead of theme.spacing for new components

**Review by sk1e — CHANGES_REQUESTED** [review]

File: src/app/pages/CreditPool/Intro/Intro.tsx [hunk 3]
```diff
@@ -26,7 +28,18 @@ export function CreditPoolIntro() {
...
+          fullWidth
+          size="large"
+          variant="contained"
+          color="primary"
+          component={RouterLink}
+          to={routes.stats.getRedirectPath()}
+        >
+          Mainnet
+        </Button>
+        {/* <Button
```
Comment (in19farkt) [inline src/app/pages/CreditPool/Intro/Intro.tsx]: Remove comment

File: src/components/Layout/Layout.style.ts [hunk 4]
```diff
@@ -1,149 +1,145 @@
...
-    root: {
-      display: 'flex',
-      position: 'relative',
-      flexDirection: 'column',
-      margin: '0 auto',
-      height: '100%',
-      maxWidth: theme.breakpoints.values.desktopXL,
-      justifyContent: 'space-between',
+export const useStyles = makeStyles(theme => ({
+  root: {
```
Comment (in19farkt) [inline src/components/Layout/Layout.style.ts]: Remove unused classes

File: src/services/theme/ThemeButton/ThemeButton.style.ts [hunk 5]
```diff
@@ -28,8 +28,7 @@ export const useStyles = makeStyles(theme => ({
     bottom: 2,
     left: 2,
     borderRadius: theme.spacing(2),
-    backgroundColor:
-      theme.palette.type === 'light' ? theme.colors.athensGray : theme.colors.obsidian,
+    backgroundColor: theme.palette.type === 'light' ? theme.colors.athensGray : theme.colors.obsidian,
```
Comment (in19farkt) [inline src/services/theme/ThemeButton/ThemeButton.style.ts]: prettier error

**Review by in19farkt — APPROVED** [review]

**Pull request merged**